tests/test_slurm_minimal.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
```

Comment:
Add tests that cover all OS distributions and all instance types.

Wait for cluster to be available and then run tests on the clusters to make sure that it worked.